vue-app/src/components/RoundInformation.vue

```diff
@@ -25,7 +25,26 @@
           </div>
         </div>
         <div
+          class="round-info-item"
           v-if="
+            $store.getters.isRoundJoinPhase &&
+            !$store.getters.isRoundContributionPhase
```

Comment:
I had to add this extra condition (`!$store.getters.isRoundContributionPhase`) because I found that sometimes both `isRoundJoinPhase` and `isRoundContributionPhase` were true.

@samajammin perhaps we want to use `isRoundJoinOnlyPhase` instead of `isRoundJoinPhase`. Let me know what do you think.

Reply:
> I had to add this extra condition (!$store.getters.isRoundContributionPhase) because I found that sometimes both isRoundJoinPhase and isRoundContributionPhase were true.

Correct, at least for now it's possible for the round to be both join phase & contribution phase.

> @samajammin @wackerow perhaps we want to use isRoundJoinOnlyPhase instead of isRoundJoinPhase. Let me know what do you think.

Yep, we should just use `isRoundJoinOnlyPhase`.

Reply:
Adding this now...

Reply:
Great! thanks for the explanation